consensus/src/dag/bootstrap.rs

```diff
@@ -416,15 +425,57 @@ impl DagBootstrapper {
             metadata_adapter,
             heuristic,
             100,
-            ChainHealthBackoffConfig::new(self.config.chain_backoff_config.clone()),
         ))
     }
 
+    fn build_anchor_election(
+        &self,
+    ) -> (
+        Arc<dyn AnchorElection>,
+        Arc<dyn CommitHistory>,
+        Option<Vec<CommitEvent>>,
```

Comment:
roughly remember we discussed this, the abstraction here looks weird. I'd imagine we have a concrete `CommitHistory` struct that can return CommitEvent, and AnchorElection can use the CommitHistory (impl MetadataBackend) internally for reputation.

Reply:
discussed offline, will follow-up